cmd/genji/shell/shell.go

```diff
@@ -256,9 +257,35 @@ func (sh *Shell) getDB() (*genji.DB, error) {
 		os.Exit(2)
 	}
 
+	err = sh.runPipedInput()
```

Comment:
I don't think we should run this here but rather on L93-94. Basically, there are two modes:
- if we find something on stdin: exec the query, display the output, stop the program
- else, we run the shell

```bash
$ echo "CREATE TABLE foo; INSERT INTO foo (a) VALUES (1); SELECT * FROM FOO" | genji 
{
  "a": 1
}
$ ...
```